### DISCUSSION TITLE
Migration to Independent Org for  DnD App

### DISCUSSION THREAD

**Discussion by WilkinsonK** [wilkinson-workshop/dnd-app#87] [Ideas]
# Intent
Migration of our application to my general org, wilkinson-workshop, is a temporary one. In the meantime, we need to workshop a few things. These items can be discussed here or in other forums.
The reasoning here is I don't want to pollute our project boards with SPIKEs all over the place when these need to be ideas that are shared across the org.

The TODOs that we have to go over are, but not limited to:
- [ ] A name for our application
- [ ] A name for our organization

@joshuahoff1 You are welcome to comment/add to this discussion.

# Organization Name
We need name for the organization. This needs to be first and foremost as this will inform the nature of our work and the direction we would like to head in.

The criteria we need to meet
---

1. Must be distinct. Something that stands out against other orgs/competitors.
2. Must be cohesive with our combined vision.
3. Must send a message. While we want to work as a recreational group, the name is the summary of our mission statement.

**Comment by WilkinsonK** [comment]
Some names that might fit our criteria:
1. Hoff & Wilkinson
2. Harmony
3. Goal line
4. Stack Wizard
5. Thomas, Richard & Harold
6. Dreamable
7. Inventure
8. Bloom (Inventive|Creative|Agent|Engineering|...)

**Comment by WilkinsonK** [comment]
Potential name(s) for our application:
1. Combat Wizard
2. Combat Companion

**Reply by joshuahoff1** [reply]
I like combat companion the most.
